src/Attachment.tsx

```diff
@@ -33,6 +33,8 @@ const queryString = (query: QueryParams) =>
     .map(key => encodeURIComponent(key) + '=' + encodeURIComponent(query[key].toString()))
     .join('&');
 
+const isNil = (value: any) => value === null || typeof value === "undefined"
```

Comment:
I hear that, but I prefer my alternative which reads more clearly by avoiding a double negative in usage. (I think there is a place for both `isNil` and `exists`, but here the only usages are to check for existence so that's the one we should use).

This is the rare issue that deals directly with existential angst.